x/slashing/simulation/invariants.go

```diff
@@ -1,14 +1,14 @@
 package simulation
 
 import (
-	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/mock/simulation"
 )
 
 // TODO Any invariants to check here?
 // AllInvariants tests all slashing invariants
 func AllInvariants() simulation.Invariant {
-	return func(_ *baseapp.BaseApp) error {
+	return func(_ sdk.Context) error {
```

Comment:
is there an issue open for slashing invariants here?

Reply:
https://github.com/cosmos/cosmos-sdk/issues/2942